engines/wintermute/base/gfx/osystem/base_surface_osystem.cpp

```diff
@@ -274,6 +273,17 @@ uint32 BaseSurfaceOSystem::getPixelAt(Graphics::Surface *surface, int x, int y)
 	return 0;
 }
 
+//////////////////////////////////////////////////////////////////////////
+bool BaseSurfaceOSystem::getPixel(int x, int y, byte *r, byte *g, byte *b, byte *a)
+{
```

Comment:
Wrong brace style, please use the same style you see in line 242.

Reply:
Fixed in b956156.